llpc/builder/llpcBuilderImplImage.cpp

```diff
@@ -1185,6 +1185,1185 @@ Value* BuilderImplImage::CreateImageStore(
+
+        // Sample BL
+        coordsChromaBL.push_back(CreateFDiv(pSubI, pWidth));
+        coordsChromaBL.push_back(CreateFDiv(CreateFAdd(pSubJ, ConstantFP::get(getFloatTy(), 1.0)), pHeight));
+        pSampleInfo->pImageDesc = xyChromaInfo.pImageDesc1;
+        Value* pBLb = static_cast<Instruction*>(YCbCrCreateImageSampleInternal(coordsChromaBL, pSampleInfo));
+        pSampleInfo->pImageDesc = xyChromaInfo.pImageDesc2;
+        Value* pBLr = static_cast<Instruction*>(YCbCrCreateImageSampleInternal(coordsChromaBL, pSampleInfo));
+        Value* pBL = CreateShuffleVector(pBLr, pBLb, { 0, 6});
+
+        // Sample BR
+        coordsChromaBR.push_back(CreateFDiv(CreateFAdd(pSubI, ConstantFP::get(getFloatTy(), 1.0)), pWidth));
+        coordsChromaBR.push_back(CreateFDiv(CreateFAdd(pSubJ, ConstantFP::get(getFloatTy(), 1.0)), pHeight));
+        pSampleInfo->pImageDesc = xyChromaInfo.pImageDesc1;
+        Value* pBRb = static_cast<Instruction*>(YCbCrCreateImageSampleInternal(coordsChromaBR, pSampleInfo));
+        pSampleInfo->pImageDesc = xyChromaInfo.pImageDesc2;
+        Value* pBRr = static_cast<Instruction*>(YCbCrCreateImageSampleInternal(coordsChromaBR, pSampleInfo));
+        Value* pBR = CreateShuffleVector(pBRr, pBRb, { 0, 6});
+
+        // Linear interpolate
+        pResult = BilinearBlend(pAlpha, pBeta, pTL, pTR, pBL, pBR);
+    }
+
+    return pResult;
+}
+
+// =====================================================================================================================
+// Create YCbCr image sample internal
+Value* BuilderImplImage::YCbCrCreateImageSampleInternal(
+    SmallVectorImpl<Value*>& coords,     // ST coords
+    YCbCrSampleInfo*         pYCbCrInfo) // [In] YCbCr smaple information
+{
+    Value* pCoords = CreateInsertElement(Constant::getNullValue(VectorType::get(coords[0]->getType(), 2)),
+                                         coords[0],
+                                         uint64_t(0));
+
+    pCoords = CreateInsertElement(pCoords, coords[1], uint64_t(1));
+
+    return CreateImageSampleGather(pYCbCrInfo->pResultTy,
+                                   pYCbCrInfo->dim,
+                                   pYCbCrInfo->flags,
+                                   pCoords,
+                                   pYCbCrInfo->pImageDesc,
+                                   pYCbCrInfo->pSamplerDesc,
+                                   pYCbCrInfo->address,
+                                   pYCbCrInfo->instName,
+                                   pYCbCrInfo->isSample);
+}
+
+// =====================================================================================================================
+// Replace [beginBit, beginBit + adjustBits) bits with data in specific word
+Value* BuilderImplImage::ReplaceBitsInWord(
+    Value*      pWord,      // [in] Target wowrd
+    uint32_t    beginBit,   // The first bit to be replaced
+    uint32_t    adjustBits, // The number of bits should be replaced
```

Comment:
Consider rewording this to better reflect that it's just a copy operations. For example, `s/adjustBits/numBitsToCopy` or `numBits` or `bitLen`, etc.